contrib/idxd-config-cmake/CMakeLists.txt

```diff
@@ -8,16 +8,16 @@ set (SRCS
     "${LIBACCEL_SOURCE_DIR}/util/sysfs.c"
 )
 
-add_library(accel-config ${SRCS})
+add_library(_accel-config ${SRCS})
```

Comment:
Why?

Reply:
Just want to align with [CMakeLists.txt in master](https://github.com/ClickHouse/ClickHouse/blob/master/contrib/idxd-config-cmake/CMakeLists.txt)